library/build.gradle

```diff
@@ -134,7 +134,7 @@ dependencies {
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
     testImplementation "org.json:json:20190722"
     testImplementation "androidx.arch.core:core-testing:2.1.0"
-    testImplementation 'org.amshove.kluent:kluent-android:1.61'
+    testImplementation 'org.amshove.kluent:kluent:1.61'
```

Comment:
Is it better than Truth from google or AssertJ?
Among 3 libraries we already have 3 assertion libraries and that's just not very convenient to switch between them.

Reply:
Kluent is convenient to use in Kotlin tests because of its concise dsl. It is an extension. Under the hood, it uses `kotlin.test` library for assertions. 

Imo, Truth/AJ is a good choice for Java, and it is worth to try some more convenient syntax in Kotlin.

Reply:
Have concerns because it seems a pet project of guy and not very popular. And in the worst case some assertion might miss requirement. So it's in the end `safety vs nice looking api`.
But ok, let's try it. I'll try it on llc as well. We can evaluate later whats fits better our projects.

Reply:
Alright 👍